charts/galley/values.yaml

```diff
@@ -93,6 +93,12 @@ config:
         defaults:
           lockStatus: unlocked
           status: enabled
+      enforceFileDownloadLocation:
+        defaults:
+          lockStatus: locked
+          status: disabled
+          config:
+            enforcedTimeoutSeconds: ""
```

Comment:
why not `0`?

Reply:
Oops, that's the wrong config (I must have copy-pasted from `selfDeletingMessages`). I've turned it into an empty config now.